pom.xml

```
@@ -357,7 +363,7 @@
             <dependency>
                 <groupId>nl.b3p</groupId>
                 <artifactId>kadaster-gds2</artifactId>
-                <version>2.6</version>
+                <version>2.7-SNAPSHOT</version>
```

gisteren is op verzoek 2.7 gemaakt, maar die zit er nog niet in?...
```suggestion
                <version>2.7</version>
```